app/extensions.js

```diff
@@ -394,10 +393,6 @@ module.exports.init = () => {
     }
   })
 
-  ipcMain.on(messages.LOAD_URL_REQUESTED, (e, tabId, url) => {
```

Comment:
Curious why this was removed? What is/was emitting this?

Reply:
@bsclifton Good question! This is actually being emitted in PDF.JS: https://github.com/brave/pdf.js/blob/master/extensions/chromium/pdfHandler.js#L136. However I moved all of that logic into this PR [1], so it no longer needs to be handled in browser-laptop with this PR.

[1] Technically all of this logic is still in the PDF.JS extension since we can't update browser-laptop and PDF.JS at the same time. However the manifest override in line 452 means that once someone updates to this version of browser-laptop, the duplicated code in PDF.JS will no longer run.

Reply:
Ah- perfect 😄  Thanks for the explanation!